src/app/zap-templates/templates/app/attributes/Accessors-src.zapt

```diff
@@ -25,15 +25,18 @@ namespace Attributes {
 {{/first}}
 {{#if clusterRef}}
 {{#if (canHaveSimpleAccessors type)}}
-EmberAfStatus Get{{asUpperCamelCase label}}(chip::EndpointId endpoint, {{asUnderlyingZclType type}} * {{asLowerCamelCase label}})
+{{#if_is_struct type}}
```

Comment:
You should not need has `canHaveSimpleAccessors` should already check if this is not a struct.